config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :dishes, only: :show
+  post "/dishes/:id/ingredients", to: "dishes/ingredients#create"
```

Comment:
Totally fine to use handrolled routes here! Nice work using RESTful routing here regardless! Another option would be to use a DishIngredients controller instead of namespacing it under dishes (but that's fine, too)

If you had put `resources :ingredients, only: :create` nested under dishes, you would've maybe had to add the `controller: "dishes/ingredients` if it didn't default for you -- but not sure that was the error you were running into!